app/src/test/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/ImportLoginsScreenTest.kt

```diff
@@ -369,7 +361,17 @@ class ImportLoginsScreenTest : BaseComposeTest() {
             .filterToOne(hasAnyAncestor(isDialog()))
             .assertIsDisplayed()
             .performClick()
-        verifyActionSent(ImportLoginsAction.FailSyncAcknowledged)
+        verifyActionSent(ImportLoginsAction.FailedSyncAcknowledged)
+    }
+
+    @Test
+    fun `Success bottom sheet is shown when state is updated`() {
+        mutableImportLoginsStateFlow.tryEmit(
+            DEFAULT_STATE.copy(showBottomSheet = true),
+        )
+        composeTestRule
+            .onNodeWithText("Import Successful!")
+            .assertIsDisplayed()
     }
```

Comment:
For some reason verifying `viewModel.trySendAction(SuccessfulSyncAcknowledged)` was able to successfully `performClick()` on the correct nodes and also tried with `.performSemanticsAction(SemanticsActions.OnClick)` but neither seemed to invoke the lambda in the test and it  _is_ working in the app.